modules/installation-and-upgrade/pages/container-management/updating-proxy-containers.adoc

```diff
@@ -1,6 +1,16 @@
 = Update containers
 
-The {productname} {productnumber} Proxy containers running on [literal]``podman`` can be updated using the following command:
+Before running the upgrade command, it is recommended to upgrade the [literal]``mgrpxy`` tool first.
+
+. One can do so by running the following command:
++
+----
+transactional-update
+----
+
+. If updates were applied, [literal]``reboot``.
+
+. The {productname} {productnumber} Proxy containers running on [literal]``podman`` can be updated using the following command:
```

Comment:
```suggestion
. The {productname} {productnumber} Proxy containers running on [literal]``podman`` can be updated using the following command:
+

```
Without this "connector" adoc tools sometimes end the procedure the following text block.